### PR TITLE
KIE forms jBPM integration:

### PR DESCRIPTION
**Pull request by pefernan**
- Support for BeanValidation for dynamic models
    - Fixing minor issues

KIE forms jBPM integration:
    - Fixing binding strategy for jBPM

Merge with: https://github.com/droolsjbpm/kie-wb-common/pull/572

### CODE REVIEW

**Comment by pefernan** [comment]
jenkins retest this

**Comment by pefernan** [comment]
jenkins retest this

File: jbpm-console-ng-workbench-forms-integration/jbpm-console-ng-workbench-forms-integration-backend/src/main/java/org/jbpm/console/ng/workbench/forms/display/backend/provider/TaskFormValuesProcessor.java [hunk 1]
```diff
@@ -52,66 +48,46 @@ protected String getFormName( TaskRenderingSettings settings ) {
...
 
     @Override
-    protected Map<String, Object> getOutputValues( Map<String, Object> values, FormDefinition form ) {
-        Map<String, Object> result = new HashMap<>();
-
+    protected Map<String, Object> getOutputValues( Map<String, Object> values, FormDefinition form, TaskRenderingSettings settings ) {
         if ( isValid( form ) ) {
-
-            TaskFormModel model = (TaskFormModel) form.getModel();
+            /*TaskFormModel model = (TaskFormModel) form.getModel();
```
Comment (cristianonicolai) [inline jbpm-console-ng-workbench-forms-integration/jbpm-console-ng-workbench-forms-integration-backend/src/main/java/org/jbpm/console/ng/workbench/forms/display/backend/provider/TaskFormValuesProcessor.java]: perhaps remove comment out code?

**Comment by pefernan** [comment]
@cristianonicolai Added more changes... Can you review it again?

**Comment by cristianonicolai** [comment]
@pefernan sorry for the confusion here, somehow I though the dependent PR (https://github.com/droolsjbpm/kie-wb-common/pull/572) was merged, when is clearly not. Would you mind reopening this? Thanks.

**Comment by pefernan** [comment]
Ok, let's try.

**Pull request merged**